src/simulation/monsters/low/g-m/Hydra.ts

```diff
@@ -6,18 +6,19 @@ import { GemTable } from '../../../subtables/RareDropTable';
 import RareSeedTable from '../../../subtables/RareSeedTable';
 
 const HydraOffTaskUniqueTable = new LootTable()
-	.oneIn(1_801, "Hydra's eye")
-
-	.oneIn(5_001, 'Hydra tail')
 	.oneIn(10_000, 'Dragon thrownaxe', [200, 400])
-	.oneIn(10_001, 'Dragon knife', [200, 400]);
+	.oneIn(10_000, 'Dragon knife', [200, 400])
+	.oneIn(5_000, 'Hydra tail')
+	.oneIn(1_800, "Hydra's eye")
+	// so they show up on the drop table:
+	.oneIn(1_000_000, "Hydra's fang")
+	.oneIn(1_000_000, "Hydra's heart");
```

Comment:
lol you cant do this, you cant add fake drops to a monster just so it shows up in the cl command.